### PR TITLE
feature: add pull commands using buildah #986

### PR DESCRIPTION
**Pull request by yyf1986**
[SKIP CI]seaols: 一句话简短描述该PR内容
feature: add pull commands using buildah #986

### CODE REVIEW

File: cmd/sealos/cmd/pull.go [hunk 1]
```diff
@@ -37,6 +38,9 @@ func newPullCmd() *cobra.Command {
 }
 
 func init() {
+	if buildah.InitReexec() {
```
Comment (cuisongliu) [inline cmd/sealos/cmd/pull.go:41]: buildah init  can move to servic interface new func

File: pkg/image/service.go [hunk 2]
```diff
@@ -35,7 +36,15 @@ func NewRegistryService() (types.RegistryService, error) {
 	if err := checkBuildah(); err == nil {
 		return binary.NewRegistryService()
 	}
-	return nil, errors.New("buildah not found in system path")
+	err := buildahPolicySync()
```
Comment (cuisongliu) [inline pkg/image/service.go:39]: if sdk and binary need this func ,we can init it at New Service func .

File: cmd/sealos/cmd/pull.go (hunk 1 above)
Comment (fanux) [reply cmd/sealos/cmd/pull.go:41]: main func already has call this func?

File: pkg/image/buildah/registry/registry.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2022 191557539@qq.com.
```
Comment (fanux) [inline pkg/image/buildah/registry/registry.go:2]: 191557539@qq.com -> sealos

File: cmd/sealos/cmd/pull.go (hunk 1 above)
Comment (yyf1986) [reply cmd/sealos/cmd/pull.go:41]: I fix it.this func already has call.

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**